src/gui/vector/qgsvectorlayerproperties.cpp

```diff
@@ -136,18 +136,17 @@ QgsVectorLayerProperties::QgsVectorLayerProperties(
   mActionLoadStyle = menuStyle->addAction( tr( "Load Style…" ) );
   connect( mActionLoadStyle, &QAction::triggered, this, &QgsVectorLayerProperties::loadStyle );
 
+  mActionSaveStyle = menuStyle->addAction( "Save Current Style…" );
```

Comment:
```suggestion
  mActionSaveStyle = menuStyle->addAction( tr( "Save Current Style…" ) );
```

Reply:
:+1:  thank you for the review, I missed this one !